src/bpffeature.cpp

```diff
@@ -642,6 +642,12 @@ std::string BPFfeature::report(void)
       << "  raw_tp_special: " << to_str(has_raw_tp_special())
       << "  iter: " << to_str(has_iter("task")) << std::endl;
 
+#ifdef HAVE_LIBSYSTEMD
+  buf << "Systemd support: yes" << std::endl;
+#else
+  buf << "Systemd support: no" << std::endl;
+#endif
+
```

Comment:
This file is meant to be just for reporting BPF functionality detected at runtime. We've got a separate file for reporting build information: [src/build_info.cpp](https://github.com/bpftrace/bpftrace/blob/14cb69defaaf047f63c9c5fa21be27cd446f9315/src/build_info.cpp)